libs/community/tests/integration_tests/chat_models/test_litellm_router.py

```diff
@@ -172,17 +212,16 @@ def litellm_router() -> Any:
     """LiteLLM router for testing."""
     from litellm import Router
 
-    return Router(
-        model_list=model_list,
-    )
+    return Router(model_list=model_list)
 
 
 @pytest.mark.scheduled
+@pytest.mark.enable_socket
```

Comment:
We don't run community integration tests in CI. is this needed?

Reply:
pytest.mark.enable_socket is needed to run the tests manually on a dev machine, since later versions of pytest disable sockets by default, and we need sockets enabled to run these tests.